packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -23,19 +45,56 @@ const useContentStyles = makeStyles({
   selected: typographyStyles.body1Strong,
 });
 
+const useIndicatorStyles = makeStyles({
+  base: {
+    '::after': {
+      position: 'absolute',
+      transform: `translateX(${navItemTokens.indicatorOffset})`, // per spec
```

Comment:
based on the comment above, this would need to be changed to the following:
```suggestion
      transform: `translateX(-${navItemTokens.indicatorOffset})`, // per spec
```

Reply:
You should make sure this transform works correctly in RTL. You might need to explicitly handle that case.

Reply:
You are correct in that there are gremlins here. Tab uses grid instead of flexbox and that seems to handle the issue. Will likely align.

Reply:
updated and addressed the RTL issues :-)